index.html

```diff
@@ -58,9 +70,16 @@ <h2 id="project_tagline"></h2>
 <hr>
 
 <h2>
+<a id="usage-common" class="anchor" href="#usage-common" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Usage Common</h2>
+
+  <p>REMREM Common is a common repository of codes which helps both CLI and service to validate eiffel messages and publish them on RabbitMQ. It cannot be run as micro services or stand alone versions. It just help in proper working of both CLI and service. </p>
```

Comment:
I think it should be made clear, we are talking about publish? REMREM Common can be changed to publish common?